### DISCUSSION TITLE
Suggestion: waypoints, stations, depots etc. without "RailURI" as optional

### DISCUSSION THREAD

**Discussion by StolleJay** [voxeltycoon/issues#455] [Modding]
Please rework the system so you can place waypoints, stations, depots etc. over/beside the rails without to define which rail it uses so you can place as ex. a waypoint on multiple different rail types instead of have for each rail type a separate waypoint object.

**Comment by victorleblais** [comment]
It would require a way to know "where the tracks go", especially since you can do custom spacing with the .station files. 

And you also don't necessarily want to have waypoint/station that works will all types of tracks too.

It's not as simple as it seems. I'm not saying it's impossible, of course, but it requires some thinking to come up with an elegant solution that isn't overcomplicated.

**Comment by StolleJay** [comment]
> And you also don't necessarily want to have waypoint/station that works will all types of tracks too.

Because of that i wrote "as optional" so if you dont specify directly for what track it is then you can place it on every track type.
